app/src/main/java/com/willowtree/vocable/MainActivity.kt

```diff
@@ -34,11 +29,11 @@ class MainActivity : BaseActivity() {
             .replace(R.id.fragment_container, PresetsFragment())
             .commit()
 
-        keyboard_button.action = {
-            loadFragment(KeyboardFragment())
-        }
-
         presetsViewModel = ViewModelProviders.of(this).get(PresetsViewModel::class.java)
+
+        fragment_container.addOnLayoutChangeListener { _, _, _, _, _, _, _, _, _ ->
```

Comment:
Have y'all thought about migrating to ViewBinding now that 3.6 is stable, versus synthetics?

Reply:
We'll look into doing this with some of our future work 👍